index.html

```diff
@@ -0,0 +1,126 @@
+                <div class="paragraph">
+                    <p class="paragraph-p">
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatum obcaecati expedita velit dolorem, impedit praesentium ipsum unde, tempore id itaque voluptas maiores quod deserunt vero ipsa fuga tempora enim! Mollitia.
+                    </p>
+                    <h1 class="year-h1">2022.10.15(THU) ~ 16(FRI)</h1>
+                    <p class="paragraph-p-1">@ Addis Ababa bole sebategna sementegna zetenegna aseregna</p>
+                </div>
+            </div>
+        </section>
+    </section>
+    <section class="main-program">
+        <p class="main-program-p">Main Program</p>
+        <hr class="h3-hr">
+        <div class="program-container">
+        <div class="program program1">
+            <img class="program-item" src="./img/opening.png" alt="">
+            <h3 class="program-item h3-program">Opening</h3>
+            <p class="program-item">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Natus necessitatibus commodi at! Rem, accusamus reprehenderit.</p>
+        </div>
+        <div class="program program1">
+            <img class="program-item" src="./img/concert-1.png" alt="">
+            <h3 class="program-item h3-program">Songs</h3>
+            <p class="program-item">Lorem ipsum dolor sit amet consectetur adipisicing elit. Commodi adipisci corrupti architecto voluptatum provident saepe.</p>
+        </div>
+        <div class="program program1">
+            <img class="program-item" src="./img/pulpit.png" alt="">
+            <h3 class="program-item h3-program">Preaching</h3>
+            <p class="program-item">Lorem ipsum dolor sit amet consectetur adipisicing elit. Cum iste, quas excepturi modi alias delectus.</p>
+        </div>
+        <div class="program program1">
+            <img class="program-item" src="./img/closed.png" alt="">
+            <h3 class="program-item h3-program">Closing</h3>
+            <p class="program-item">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ratione recusandae velit quia, temporibus quidem tempore?</p>
+        </div>
+        <div class="program program1">
+            <img class="program-item" src="./img/photo.png" alt="">
+            <h3 class="program-item h3-program">Photo</h3>
+            <p class="program-item">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ut obcaecati provident ipsum tenetur itaque dignissimos!</p>
+        </div>
+      </div>
+        <div class="join">
+            <a class="join-button" href="#">Join Zema Songs Concert 2022</a>
+            <a class="seeall-link" href="">SEE The WHOLE PROGRAM</a>
+        </div>
+    </section>
+
+    <section class="performers">
+        <h3 class="title-speakers">Featured Speakers</h3>
+        <hr class="h3-hr">
+        <div class="flex-12">
+        <div class="performers1"></div>
```

Comment:
- [ ] In the mobile view. There is a design implementation that we still have not considered yet. Take a look 👇🏼:

  | Design | Implementation |
  | --- | --- |
  | ![Screenshot 2022-06-30 193900](https://user-images.githubusercontent.com/45002654/176736923-f2844dda-dd9f-484d-89e5-cd5beb25db9a.png) | ![image](https://user-images.githubusercontent.com/45002654/176736381-df7f55a3-121b-40fc-866e-173f5273b568.png) |
  
  As can be realized above, the design of this section in the mobile view only displays two speakers. The rest can be viewed by clicking on the button provided just after the two speakers.
  
  - [ ] Please refactor the implementation of this section's design.
  - [ ] Please add the `MORE` button. It should only be visible in the mobile view.

Reply:
@Lameck1 please this is optional to add the button it's not required on the project

Reply:
@EyasuTeshome, my bad. You are totally right, I beg your pardon for not realizing this.